bigquery/src/main/java/com/google/cloud/bigquery/samples/ExportDataCloudStorageSample.java

```diff
@@ -42,20 +42,17 @@ protected ExportDataCloudStorageSample() {
    * @throws InterruptedException Should never be thrown.
    */
   // [START main]
-  public static void main(final String[] args)
-      throws IOException, InterruptedException {
+  public static void main(final String[] args) throws IOException, InterruptedException {
     Scanner scanner = new Scanner(System.in);
```

Comment:
I'm sure there is a good reason why we don't use command line args, w/ reasonable defaults.